### PR TITLE
gui/lang: Update more lang strings.

### PR DESCRIPTION
**Pull request by Croden1999**
# About
- update lang strings for the pkg/archive Install dialog, Content Manager, Settings, the “Version” of Update History (i did not sort it) etc., some language are based on psvita, and also fix some lang.
- fixed the wrong strings in Settings.

# Result
- Content Manager
image
image
image
image

- Home Screen button
image
image

- Install Dialog
image
image
image
image
image

- Settings (Theme & Home Screen Background)
image
image

### CODE REVIEW

**Comment by Croden1999** [comment]
I want this PR to a draft, plz, sir.

File: lang/en.xml [hunk 1]
```diff
@@ -196,8 +196,26 @@ Do not power off the system or close the application.</warning_saving>
...
+		<select_directory>Select Directory</select_directory>
+		<compatible_content>archive(s) found with compatible contents.</compatible_content>
+		<successed_install>contents of this archive have successed installed:</successed_install>
+		<update_app>Update App to:</update_app>
+		<failed_install>content have failed installed:</failed_install>
+		<not_compatible_content>archive(s) did no found compatible contents:</not_compatible_content>
+		<delete_archive>Delete archive?</delete_archive>
 		<install_failed>Could not install.</install_failed>
 		<install_complete>Installation complete.</install_complete>
+		<installin>Installing</installin>
```
Comment (Zangetsu38) [inline lang/en.xml:218]: What is that ?

File: lang/fr.xml [hunk 2]
```diff
@@ -183,6 +183,7 @@ Ne pas éteindre le système, ni fermer l'application.</warning_saving>
 		<notif_deleted>Les notifications vont être supprimées.</notif_deleted>
 		<install_failed>L'installation a échoué.</install_failed>
 		<install_complete>Installation terminée.</install_complete>
+		<installin>Installation en cours</installin>
```
Comment (Zangetsu38) [inline lang/fr.xml:186]: Why you duplicate ?

Comment (Croden1999) [reply lang/fr.xml:186]: I just want to separate, if want to unify i can be restored.

Comment (Zangetsu38) [reply lang/fr.xml:186]: i no sur psvita have this string 🤔

Comment (Croden1999) [reply lang/fr.xml]: > i no sur psvita have this string 🤔

i restore it.

**Comment by Croden1999** [comment]
I'm ready, Mr. @Zangetsu38 .

File: lang/en-gb.xml [hunk 3]
```diff
@@ -39,7 +39,7 @@
 		<application>Application</application>
 		<addcont>DLCs</addcont>
 		<license>License</license>
-		<save_data>Save Data</save_data>
+		<save_data>Saved Data</save_data>
```
Comment (Zangetsu38) [inline lang/en-gb.xml:42]: is save data 🤔

File: lang/de.xml [hunk 4]
```diff
@@ -50,9 +50,10 @@
 		<delete>Löschen</delete>
 		<file_corrupted>Die Datei ist beschädigt.</file_corrupted>
 		<no>Nein</no>
-		<select_all>Wähle alle aus</select_all>
-		<select>Wählen</select>
-		<please_wait>Bitte warten...</please_wait>
+		<select_all>Alle auswählen</select_all>
+		<clear_all>Alle löschen</clear_all>
+		<select>Auswählen</select>
+		<please_wait>Warte bitte ...</please_wait>
```
[the remark this answers is not in the document]
Comment (Croden1999) [reply lang/de.xml:56]: i check my psvita again it is right.

File: lang/pt-br.xml [hunk 5]
```diff
@@ -26,16 +26,16 @@
...
+		<app_delete>Esse aplicativo e todos os dados relacionados, inclusive dados salvos, serão excluídos.</app_delete>
+		<save_delete>Deseja excluir esses dados salvos?</save_delete>
+		<eligible>Qualificado</eligible>
+		<ineligible>Não qualificado</ineligible>
 		<level>Nível</level>
 		<name>Nome</name>
-		<trophy_earning>Ganho de Troféus</trophy_earning>
-		<parental_Controls>Controle dos Pais</parental_Controls>
+		<trophy_earning>Conquista de troféus</trophy_earning>
+		<parental_Controls>Controle parental</parental_Controls>
```
Comment (Zangetsu38) [inline lang/pt-br.xml:38]: same before, text is fine 🤔, i have good set it

File: lang/pt-br.xml [hunk 6]
```diff
@@ -3,21 +3,21 @@
...
+			<last_apps_used>Últimos apps usados</last_apps_used>
 		</emulation>
 		<configuration name="Configuração">
 			<settings>Configurações</settings>
-			<user_management>Gerenciamento dos Usuários</user_management>
+			<user_management>Gerenciamento dos usuários</user_management>
 		</configuration>
 		<controls name="Controles">
-			<keyboard_controls>Controles do Teclado</keyboard_controls>
+			<keyboard_controls>Controles do teclado</keyboard_controls>
```
Comment (Zangetsu38) [inline lang/pt-br.xml:20]: revert file menu change, Caps is normal

File: lang/pt-br.xml [hunk 7]
```diff
@@ -46,12 +46,13 @@
...
-		<delete>Apagar</delete>
+		<delete>Excluir</delete>
 		<file_corrupted>O arquivo está corrompido.</file_corrupted>
 		<no>Não</no>
-		<select_all>Selecionar Tudo</select_all>
+		<select_all>Selecionar tudo</select_all>
+		<clear_all>Limpar tudo</clear_all>
 		<select>Selecionar</select>
-		<please_wait>Por favor aguarde...</please_wait>
+		<please_wait>Aguarde...</please_wait>
```
Comment (Zangetsu38) [inline lang/pt-br.xml:55]: same os already fine too here

File: lang/pt-br.xml [hunk 8]
```diff
@@ -124,17 +125,31 @@
...
+		<one_hour_ago>1 hora atrás</one_hour_ago>
+		<one_minute_ago>1 minuto atrás</one_minute_ago>
 		<bronze>Bronze</bronze>
 		<gold>Ouro</gold>
 		<platinum>Platina</platinum>
 		<silver>Prata</silver>
-		<hours_ago>{} Horas Atrás</hours_ago>
-		<minutes_ago>{} Minutos Atrás</minutes_ago>
+		<hours_ago>{} horas atrás</hours_ago>
+		<minutes_ago>{} minutos atrás</minutes_ago>
```
Comment (Zangetsu38) [inline lang/pt-br.xml:136]: same 😭 no change my offical string already set, is fine

File: lang/pt-br.xml [hunk 9]
```diff
@@ -173,7 +188,7 @@ Não desligue o sistema ou feche o aplicativo.</warning_saving>
 	</dialog>
 
 	<game_data>
-		<app_close>O seguinte aplicativo fechará.</app_close>
+		<app_close>O seguinte aplicativo será fechado.</app_close>
```
Comment (Zangetsu38) [inline lang/pt-br.xml:191]: same

File: lang/pt-br.xml [hunk 10]
```diff
@@ -201,37 +216,41 @@ O seu sistema Vita3K está pronto!</completed_setup>
 	</live_area>
 
 	<settings name="Configurações">
-		<theme_background name="Tema & Cenário de Fundo">
+		<theme_background name="Tema e fundo">
```
Comment (Zangetsu38) [inline lang/pt-br.xml:219]: ? revert

File: lang/pt-br.xml [hunk 11]
```diff
@@ -201,37 +216,41 @@ O seu sistema Vita3K está pronto!</completed_setup>
...
 			<default>Padrão</default>
 			<theme name="Tema">
-				<information name="Informação">
+				<find_in_altervista>Encontrar na Altervista</find_in_altervista>
+				<information name="Informações">
```
Comment (Zangetsu38) [inline lang/pt-br.xml:223]: no change string almeady set

File: lang/de.xml [hunk 12]
```diff
@@ -50,9 +50,10 @@
 		<delete>Löschen</delete>
 		<file_corrupted>Die Datei ist beschädigt.</file_corrupted>
 		<no>Nein</no>
-		<select_all>Wähle alle aus</select_all>
-		<select>Wählen</select>
-		<please_wait>Bitte warten...</please_wait>
+		<select_all>Alle auswählen</select_all>
+		<clear_all>Alle löschen</clear_all>
```
Comment (Zangetsu38) [inline lang/de.xml:54]: sorry for delay, i current checking, take time, move clear_all inside content manager, is no inside common string, is inside content manager app

File: lang/en-gb.xml [hunk 13]
```diff
@@ -192,10 +208,47 @@ Do not power off the system or close the application.</warning_saving>
...
+		<check_log>Please check log for more details.</check_log>
+		<select_install_type>Select install type</select_install_type>
+		<select_file>Select File</select_file>
+		<select_directory>Select Directory</select_directory>
+		<compatible_content>archive(s) found with compatible contents.</compatible_content>
+		<successed_install>contents of this archive have successed installed:</successed_install>
+		<update_app>Update App to:</update_app>
+		<failed_install>content have failed installed:</failed_install>
+		<not_compatible_content>archive(s) did no found compatible contents:</not_compatible_content>
+		<delete_archive>Delete archive?</delete_archive>
```
[the remark this answers is not in the document]
Comment (Zangetsu38) [reply lang/en-gb.xml:268]: thx :)

File: lang/de.xml (hunk 4 above)
[the remark this answers is not in the document]
Comment (Zangetsu38) [reply lang/de.xml:56]: sorry, you have right, no idea why some string is wrong inside lang, other user have done some but i no remmeber

File: lang/en-gb.xml (hunk 3 above)
Comment (Zangetsu38) [reply lang/en-gb.xml]: for it i have no idea i need check more 💀 i have no set this tring, so i no know

Comment (Zangetsu38) [reply lang/en-gb.xml]: is not really writed in all lang, is only debug option, so english only
image

**Comment by Zangetsu38** [comment]
yoiu have get very big works, i need this ,time serious check, your all string changed is normalmly fine, but i see some erorr string in set on wrong place

**Comment by Croden1999** [comment]
i found wrong strings in Settings - Language - Input languages, i will fix them.

File: vita3k/lang/include/lang/state.h [hunk 14]
```diff
@@ -119,6 +119,7 @@ struct LangState {
         std::map<std::string, std::string> help = {
             { "title", "Help" },
             { "about", "About" },
+            { "vita3k_update", "Vita3K Update" },
```
Comment (Zangetsu38) [inline vita3k/lang/include/lang/state.h:122]: delete it :D i already task on this 👌

File: lang/en-gb.xml [hunk 15]
```diff
@@ -22,6 +22,7 @@
 		</controls>
 		<help name="Help">
 			<about>About</about>
+			<vita3k_update>Vita3K Update</vita3k_update>
```
Comment (Zangetsu38) [inline lang/en-gb.xml:25]: all for it :)  n oneed create value one already exist :)

**Comment by Croden1999** [comment]
plz check it again, sir. :) @Zangetsu38

File: lang/pt-br.xml (hunk 6 above)
Comment (Zangetsu38) [reply lang/pt-br.xml:20]: done this please, keep caps for all word here

**Review by Zangetsu38 — APPROVED** [review]

**Pull request merged**